Hazel.UnitTests/SocketCapture.cs

```diff
@@ -172,21 +180,59 @@ private void SendToLocalLoop()
             }
         }
 
-        public void DiscardPacketForLocal()
+        public void AssertPacketsToLocal(int pktCnt)
```

Comment:
For these two Assert methods, I'd recommend changing the names - I got pretty confused by the Assert part - maybe something like `WaitForPacketsToLocalCount` or `AssertPacketsToLocalReachCount`?